docs/DeveloperGuide.md

```diff
@@ -58,7 +56,7 @@ that the user has selected to view more details about a specific feature or gene
 of Realestatepro
 
 ## Property
-The `Property` is a new attribute that can be added to a `Person` that represents a real estate property listing. A `Person` is able to hold multiple properties including none.
+The `Property` is a new attribute that can be added to a `person` that represents a real estate property listing. A `person` is able to hold multiple properties including none.
```

Comment:
Perhaps `person` should still be `Person` since we are refering to the object `Person`? Same comment for subsequent occurences